app/src/main/java/org/thoughtcrime/securesms/webrtc/CallManager.kt

```diff
@@ -721,7 +790,7 @@ class CallManager(
         connection.setCommunicationMode()
         setAudioEnabled(true)
         dataChannel?.let { channel ->
-            val toSend = if (!_videoEvents.value.isEnabled) VIDEO_DISABLED_JSON else VIDEO_ENABLED_JSON
+            val toSend = if (!_videoState.value.userVideoEnabled) VIDEO_DISABLED_JSON else VIDEO_ENABLED_JSON
```

Comment:
```suggestion
            val toSend = if (_videoState.value.userVideoEnabled) VIDEO_ENABLED_JSON else VIDEO_DISABLED_JSON
```

Reply:
d'you want to do this one? it's easier to read and reason about without the negation. It's easy to miss a `!`.